Makefile

```diff
@@ -1,11 +1,16 @@
 .PHONY: test
 test:
+	docker build -t neo-tree .
```

Comment:
Let's leave the originally `test` action alone and add the docker version as a new one called `test-docker`